auter.aptModule

```diff
@@ -63,7 +63,9 @@ function apply_updates() {
   # Set the list of debs to be installed
   if [[ "${ONLYINSTALLFROMPREP}" == "yes" ]]; then
     if [[ $(ls -1 ${DOWNLOADDIR}/${CONFIGSET}/*.deb 2>/dev/null | wc -l) -gt 0 ]]; then
-      AVAILABLE_PACKAGE_COUNT=$(${PACKAGE_MANAGER} -u --just-print install --assume-no ${PACKAGEMANAGEROPTIONS} ${DOWNLOADDIR}/${CONFIGSET}/*.deb | awk '/upgraded,.*newly installed,/ {sum=$1+$3} END {print sum}')
+      local AVAILABLE_PACKAGES=$(${PACKAGE_MANAGER} -u --just-print install --assume-no ${PACKAGEMANAGEROPTIONS} ${DOWNLOADDIR}/${CONFIGSET}/*.deb 2>&1)
+      echo "${AVAILABLE_PACKAGES}" >${DATADIR}/last-apply-output-${CONFIGSET}
+      AVAILABLE_PACKAGE_COUNT=$(echo "${AVAILABLE_PACKAGES}" | awk '/upgraded,.*newly installed,/ {sum=$1+$3} END {print sum}')
```

Comment:
Should this be declared `local` the same as the other instances this variable is used?